packages/polling-controller/src/PollingController.ts

```diff
@@ -49,19 +54,26 @@ function PollingControllerMixin<TBase extends Constructor>(Base: TBase) {
      * Starts polling for a networkClientId
      *
      * @param networkClientId - The networkClientId to start polling for
+     * @param options - The options used to group the polling events
      * @returns void
      */
-    startPollingByNetworkClientId(networkClientId: NetworkClientId) {
+    startPollingByNetworkClientId(
+      networkClientId: NetworkClientId,
+      options: object = {},
+    ) {
       const innerPollToken = random();
-      if (this.#networkClientIdTokensMap.has(networkClientId)) {
-        const set = this.#networkClientIdTokensMap.get(networkClientId);
+
+      const outerPollToken = getKey(networkClientId, options);
```

Comment:
[nit] maybe this should be called `key` of `pollingKey` or `pollingSessionKey`... I dunno. Just doesn't seem like this should be considered a pollingToken? It's confusing me hehe

Reply:
This is still called outerPollToken here?